libnetwork/internal/rootlessnetns/netns_linux.go

```diff
@@ -369,9 +369,14 @@ func (n *Netns) setupMounts() error {
 
 	// Ensure we mount private in our mountns to prevent accidentally
 	// overwriting the host mounts in case the default propagation is shared.
-	err = unix.Mount("", "/", "", unix.MS_PRIVATE|unix.MS_REC, "")
+	// However using private propagation is not what we want as new mounts/umounts
+	// are not propagted into our namespace thus we use salve.
```

Comment:
if you are going to re-push, there are a couple of typos here

Reply:
done, reworded a bit to make it more clear